Service/DataService.php

```diff
@@ -2010,4 +2010,83 @@ public function createAndMapIndex(Environment $environment): void
             'name' => $environment->getAlias()
         ]);
     }
+
+    public function lockAllContentTypesRevisions(\DateTime $until, string $by): int
+    {
+        $rows = 0;
+        $contentTypes = $this->contentTypeService->getAll();
+        foreach ($contentTypes as $contentType) {
+            $rows += $this->lockContentTypeRevisions($contentType, $until, $by);
+        }
+
+        return $rows;
+    }
+
+    public function lockContentTypeRevisions(ContentType $contentType, \DateTime $until, string $by): int
```

Comment:
I don't get the $rows variable the lockRevisions is called once so just return that result. And maybe add return type def on lockRevisions method